styles/prosilver/template/js/scoring_preview.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
In that case, take the seperate file. Doing as much as possible on server side is better and safer this way.